lib/cli.js

```diff
@@ -4,14 +4,23 @@ const { argv } = yargs
   .usage("Convert images to AVIF")
   .option("input", {
     type: "string",
-    default: "*.{jpg,jpeg,tif,tiff,webp,png,gif,svg}",
-    description: "Input file name(s), supports globs/wildcards",
+    default: process.cwd().replace(/\\/g, "/"),
+    description:
+      "Input directory, Default is the absolute path to the directory that invoked this.",
+  })
+  .alias("i", "input")
+  .option("target", {
```

Comment:
Can we use the existing `input` for this rather than adding another option? There's a lot of flexibility in the use of globs and this seems to be removing most of this.